content/rs/administering/security/account-management.md

```diff
@@ -5,62 +5,163 @@ weight: $weight
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td class="row-head">Admin</td>
+    <td class="cat-boundary" align="center" align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td class="cat-boundary" align="center" align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td class="cat-boundary" align="center" align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+  </tr>
+</table>
+
+## Adding a user
+
+To add a user to the cluster:
+
+1. Go to: **settings** > **team**
+1. Click ![Add](/images/rs/icon_add.png#no-click "Add").
+1. Enter the name, email and password of the new user and select the role to assign to the user.
+1. Select the type of user:
+    - internal - Authenticates with RS
+    - external - Authenticates with an external LDAP server
+
+      To have a user authenticate with LDAP, you must first [enable LDAP integration]({{< relref
+      "/rs/administering/security/ldap-integration.md" >}}).
+
+1. For the email alerts, click **Edit** and select the alerts that the user receives.
+    You can select:
+    - Receive alerts for databases - The alerts that are enabled for the selected databases are sent to
+      the user. You can either select all databases, or you can select **Customize** and select the
+      individual databases to send alerts for.
+    - Receive cluster alerts - The alerts that are enabled for the cluster are send to the user.
```

Comment:
1) The alerts that are enabled for the cluster (defined settings -> alerts) are sent to the user
2) I think it should be "are sent"